install.py

```diff
@@ -79,6 +79,8 @@ def install_cyclus(args):
                           ]
         if args.build_type:
             cmake_cmd += ['-DCMAKE_BUILD_TYPE=' + args.build_type]
+        else:
+            cmake_cmd += ['-DCMAKE_BUILD_TYPE=Release']
```

Comment:
Or should we set a default in the argparse declaration of this CLI parameter?

Reply:
That is a better solution, thanks for the suggestion @gonuke